### PR TITLE
Update meta.yaml

### PR DESCRIPTION
**Pull request by pgrinaway**
This is a small PR to fix the `meta.yaml` file for building conda packages.

### CODE REVIEW

File: devtools/conda-recipe/meta.yaml [hunk 1]
```diff
@@ -28,8 +28,8 @@ requirements:
     - matplotlib
     - seaborn
     - mdtraj
-    - parmed # for testing/debugging
-    #- pdbfixer
+    - parmed
```
Comment (jchodera) [inline devtools/conda-recipe/meta.yaml]: Do you need the latest parmed, or could the older parmed that comes with `ambertools` be OK?

Comment (jchodera) [reply devtools/conda-recipe/meta.yaml]: Also, you have `parmed` on line 23 as well.

Comment (jchodera) [reply devtools/conda-recipe/meta.yaml]: You can also remove `cuda92` from line 21

Comment (pgrinaway) [reply devtools/conda-recipe/meta.yaml]: I need the latest parmed. How should I specify it?

Comment (jchodera) [reply devtools/conda-recipe/meta.yaml]: For now, I'd keep `parmed` but list it last. Not sure if order helps, but it'll remind me.

Comment (pgrinaway) [reply devtools/conda-recipe/meta.yaml]: Ok, done.

**Comment by jchodera** [comment]
I'd leave out `ambertools` since `yank`, `openmoltools`, and `openmmtools` already require it, I think. And you don't use `ambertools` directly, right?

**Comment by pgrinaway** [comment]
Yeah, you're right. Removing.

**Comment by jchodera** [comment]
Go ahead and merge and release when ready

**Pull request merged**